dbt/compiled_model.py

```diff
@@ -126,8 +126,8 @@ def should_rename(self):
     def should_execute(self, args, existing):
         return True
 
-    def prepare(self, existing, target):
-        self.target = target
+    def prepare(self, existing, profile):
```

Comment:
this is a weird stateful thing that has to happen for a stupid reason. It's fine for now, but let's try to get rid of it soon